index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New-Year-party</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="main-container">
+      <section class="hello-container">
+        <div class="upper">
+          <a href="www.facebook.com" class="facebook"
+            ><img
+              src="./assets/images/facebook-1.svg"
+              alt="facebook icon"
+              class="icon1"
+          /></a>
+          <a href="www.twitter.com" class="twitter"
+            ><img
+              src="./assets/images/twitter-1.svg"
+              alt="twitter icon"
+              class="icon1"
+          /></a>
+          <a href="#" class="upper-link">English</a>
+          <a href="#" class="upper-link">My page</a>
+          <a href="#" class="upper-link">Log out</a>
+        </div>
+        <header class="hello-header">
+          <img
+            src="./assets/images/menu_black_48dp.svg"
+            alt="hamburger menu icon"
+            class="menu-image"
+          />
+          <a href="#sub-heading1" id="logo-header"
+            ><img
+              src="./assets/images/pyramid.svg"
+              alt="party logo"
+              class="party-logo1"
+          /></a>
+          <div class="menu-appear">
+            <p class="close-menu">X</p>
+            <ul class="navigation">
```

Comment:
- [ ] This is supposed to be a `navbar`, please use semantic html and wrap `navbar` content with `<nav>` tag
- [ ] Mobile navbar links  have too much space around them, the font is too big and text wraps. Please make sure that the mobile navbar looks good:
<img width="387" alt="Screenshot 2022-04-14 at 16 47 58" src="https://user-images.githubusercontent.com/77038610/163393590-b3cd5d47-b180-4a64-811d-632ff7e1f959.png">

- [ ] OPTIONAL: I strongly suggest to fix the mobile navbar, otherwise users can scroll down without closing the navbar.